jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -78,19 +82,20 @@ public Builder addEntry(Path sourceFile, AbsoluteUnixPath pathInContainer) {
      * @param sourceFile the source file to add to the layer
      * @param pathInContainer the path in the container file system corresponding to the {@code
      *     sourceFile}
-     * @param permissions the file permissions on the container. If null, then default permissions
-     *     are used (644 for files, 755 for directories)
-     * @param lastModified the modification time of the file
+     * @param permissions the file permissions on the container (Defaults to 644 for files, 755 for
```

Comment:
nit: sorry, I should have said the entire part of "Defaults to 644 ..." doesn't make much sense, because callers are always giving explicit `permissions`. There's no way to use defaults since null is not acceptable. Perhaps move this Javadoc about default permissions to the method above that doesn't take `permissions`.